### DISCUSSION TITLE
Exploring Drag and Drop with the new Angular Material CDK

### DISCUSSION THREAD

**Discussion by giscus** [timdeschryver/timdeschryver.dev#328] [Announcements]
# Exploring Drag and Drop with the new Angular Material CDK

Drag and Drop is a new feature in the new Angular Material CDK. Let’s explore it together!

https://timdeschryver.dev/blog/exploring-drag-and-drop-with-the-new-angular-material-cdk

**Comment by mhdirkse** [comment]
Dear Tim de Schrijver,

Thanks for this article about CDK drag and drop. I am going to donate. I have an update suggestion for you. You write that you need to add dependency `npm install @angular/material@7.0.0-rc.1`. This did not work for me. I needed `@angular/cdk@17.3.1`.

With kind regards,

Martijn Dirkse
mhdirkse on GitHub
www.wearefrank.nl

**Reply by timdeschryver** [reply]
Thanks Martijn! 
Keep in mind that this blog post was written when the drag and drop functionality was in release candidate. 
It might be that article is outdated here and there and that the current API is is different.

**Comment by mhdirkse** [comment]
I cannot get cdkDrop to work, only cdkDropList. I was interested in cdkDrop because I expected that I could do drag & drop between lists with replacement of the target item. Replacement instead of insertion.

**Comment by mhdirkse** [comment]
Hello Tim Deschryver,

Thanks for replying.

With kind regards,

Martijn Dirkse
